_data/people.yml

```diff
@@ -340,9 +340,18 @@ estrada_j:
   email: jorge.estrada@bsc.es
   homepage:
 
+feld_c:
+  sur_name: Feld
+  given_name: Christian
+  affiliation: bsc
+  position:
```

Comment:
Yes, do you fix it or do I?

Brian

On 2016-03-22 16:45, Robert Speck wrote:
> In _data/people.yml
> <https://github.com/JLESC/jlesc.github.io/pull/172#discussion_r57010465>:
>
>> @@ -340,9 +340,18 @@ estrada_j:
>>    email: jorge.estrada@bsc.es
>>    homepage:
>>
>> +feld_c:
>> +  sur_name: Feld
>> +  given_name: Christian
>> +  affiliation: bsc
>> +  position:
>
> He's at JSC, right..?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JLESC/jlesc.github.io/pull/172/files/03b5525669bbadb0778ec074b855d90899aafc9c#r57010465>
>



------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
Forschungszentrum Juelich GmbH
52425 Juelich
Sitz der Gesellschaft: Juelich
Eingetragen im Handelsregister des Amtsgerichts Dueren Nr. HR B 3498
Vorsitzender des Aufsichtsrats: MinDir Dr. Karl Eugen Huthmacher
Geschaeftsfuehrung: Prof. Dr.-Ing. Wolfgang Marquardt (Vorsitzender),
Karsten Beneke (stellv. Vorsitzender), Prof. Dr.-Ing. Harald Bolt,
Prof. Dr. Sebastian M. Schmidt
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------